api/v1alpha1/virtualmachine_types.go

```diff
@@ -129,6 +129,12 @@ const (
 	// or as vApp properties on an existing VM or VMTX will be set, all others will be ignored.
 	VirtualMachineMetadataOvfEnvTransport VirtualMachineMetadataTransport = "OvfEnv"
 
+	// VirtualMachineMetadataVAppConfigTransport indicates that the data set in the VirtualMachineMetadata Transport Resource,
+	// i.e., a ConfigMap or Secret, will be vApp properties on the VM, which will be exposed as vAppConfig to the Guest VM. Only properties
+	// marked userConfigurable and already present in either OVF Properties of a VirtualMachineImage
+	// or as vApp properties on an existing VM or VMTX will be set, all others will be ignored.
+	VirtualMachineMetadataVAppConfigTransport VirtualMachineMetadataTransport = "vAppConfig"
```

Comment:
A few things:

* Please try not to exceed 80 columns for comments
* Please add a note that indicates this transport does not use Guest OS customization for networking, and this transport should only be selected if the image exposes OVF properties that bootstrap the guest's networking

Reply:
Thanks! I also made changes to all other comments on metadata transport to accommodate 80 columns.

Reply:
Ha, thanks! I had done the same in a WIP branch for v1a2 which is why I just asked you to handle yours, but I should have known you would have not stopped at that :)

Reply:
haha yeah. I can't stand that they are not aligned. ;)